.github/workflows/ci.yml

```diff
@@ -18,9 +18,9 @@ jobs:
           distribution: 'temurin'
           java-version: ${{ matrix.java }}
           check-latest: true
-      - name: Build Quarkus 2.7
+      - name: Build Quarkus main
         run: |
-          git clone https://github.com/quarkusio/quarkus.git && cd quarkus && git checkout 2.7 && ./mvnw -B -s .github/mvn-settings.xml clean install -Dquickly -Prelocations -pl devtools/cli -am
```

Comment:
Not a bug, just a question:
Why did we build just a single single module `devtools/cli` (+ its dependencies)?
I would understand skipping it (`-pl !devtools/cli`), but this does not make sense to me.

Reply:
I think that was because on the previous version we were using Quarkus 2.7, so the only module that we want to compile at that time was Quarkus-cli. All the other extension was taken from MavenCentral (Quarkus 2.7)

Reply:
That sounds plausible.
But we build CLI in a later step (`PR - Linux - JVM build - Latest Version` -> `Build Quarkus CLI`).
So it seems to me like we could have skipped either this build entirely, or the `Build Quarkus CLI` step.